dash-pipeline/bmv2/stages/tunnel_stage.p4

```diff
@@ -0,0 +1,41 @@
+#ifndef _DASH_STAGE_TUNNEL_P4_
+#define _DASH_STAGE_TUNNEL_P4_
+
+control tunnel_stage(
+    inout headers_t hdr,
+    inout metadata_t meta)
+{
+    action set_tunnel_attrs(
+            @SaiVal[type="sai_ip_address_t"]
+            IPv4Address underlay_dip,
```

Comment:
I believe `dip` is good enough here, as the tunnel can be multiple tunnels, using underlay can be misleading.

Reply:
Done